src/sentry/static/sentry/app/views/eventsV2/savedQuery/index.tsx

```diff
@@ -194,9 +194,11 @@ class SavedQueryButtonGroup extends React.PureComponent<Props, State> {
   handleCreateAlertSuccess = () => {
     const {organization} = this.props;
     trackAnalyticsEvent({
-      eventKey: 'discover_v2.create_alert',
-      eventName: 'Discoverv2: Create alert from discover',
-      organization_id: parseInt(organization.id, 10),
+      eventKey: 'discover_v2.create_alert_clicked',
+      eventName: 'Discoverv2: Create alert clicked',
+      status: 'success',
+      organization_id: organization.id,
+      url: window.location.href,
```

Comment:
Not sure how helpful this URL will be since it will be pretty dynamic due to org/project slugs. You can use `app/utils/getRouteStringFromRoutes` to get a parameterized route string

Reply:
I'm not sure either, I think we'll probably end up dropping it if it isn't useful instead of using the route string as we know what page they're on already. The valuable information here would probably be the ability to admin view the exact page they were clicking from.